worlds/kh1/Options.py

```diff
@@ -155,17 +155,45 @@ class FinalRestDoor(Choice):
     option_postcards = 2
     option_superbosses = 3
 
+class RequiredPostcards(Range):
+    """
+    If your goal is set to "Postcards", defines how many postcards are needed to achieve victory.
+    """
+    display_name = "Required Postcards"
+    default = 10
+    range_start = 1
+    range_end = 10
+
+class RequiredPuppies(Choice):
+    """
+    If your goal is set to "Puppies", defines how many puppies are needed to achieve victory.
+    """
+    display_name = "Required Puppies"
+    default = 99
+    option_10 = 10
+    option_20 = 20
+    option_30 = 30
+    option_40 = 40
+    option_50 = 50
+    option_60 = 60
+    option_70 = 70
+    option_80 = 80
+    option_90 = 90
+    option_99 = 99
```

Comment:
I strongly recommend not having it default to the maximum amount. Finding 100% of mcguffins is a massive pain in my experience.

Reply:
Changed both to 8/80